config/locales/helpers/fr.yml

```diff
@@ -4,229 +4,229 @@ fr:
         tech_stack: "..."
         cta_body_markdown: "..."
-        cta_button_text: Visit Acme Site
-        cta_button_url: https://example.com
+        cta_button_text: Visiter le site Acme 
+        cta_button_url: https://votresite.com
         proof: "..."
       response_template:
         title: "..."
       user:
         name: John Doe
         email: john.doe@example.com
         username: johndoe
-        feed_url: https://yoursite.com/feed
+        feed_url: https://votresite.com/feed
     application_helper:
-      app_logo: App logo
+      app_logo: Logo de l'application
       follow:
         aria_label:
-          Organization: 'Follow organization: %{name}'
-          Tag: 'Follow tag: %{name}'
-          User: 'Follow user: %{name}'
-          default: 'Follow %{type}: %{name}'
+          Organization: "Suivre l'organisation: %{name}"
+          Tag: 'Suivre le tag: %{name}'
+          User: "Suivre L'utilisateur : %{name}"
+          default: "Suivre %{type}: %{name}"
         text:
           Organization: Suivre
           Tag: Suivre
           User: Suivre
           default: Suivre
-      required: Required
+      required: Requis
       subtitle:
-        week: Top posts this week
-        month: Top posts this month
-        year: Top posts this year
-        infinity: All posts
-        latest: Latest posts
+        week: Les meilleurs articles de cette semaine
+        month: Les meilleurs articles de ce mois
+        year: Les meilleurs articles de cette année
+        infinity: Tous les messages
+        latest: Derniers messages
       title_text: "%{title} - %{timeframe}"
     articles_helper:
       logo: "%{service_name} logo"
-      most_comments: Most Comments
-      most_reactions: Most Reactions
-      most_views: Most Views
-      recently_created: Recently Created
-      recently_published: Recently Published
+      most_comments: Le plus de commentaires
+      most_reactions: Le plus de réactions
+      most_views: Le plus de vues
+      recently_created: Récemment créé
+      recently_published: Récemment publié
     authentication_helper:
-      any_of_those: any of those
-      email_password: Email & Password
-      reminder: 'Reminder: you used %{method} to authenticate your account, so please use %{dem} to sign in if prompted.'
-      that: that
-      invite_only: You cannot do this until you disable Invite Only Mode
+      any_of_those: n'importe lequel de ces éléments
+      email_password: Email et mot de passe
+      reminder:  'Rappel : vous avez utilisé %{method} pour authentifier votre compte, veuillez donc utiliser %{dem} pour vous connecter si vous y êtes invité'
+      that: que
+      invite_only: Vous ne pouvez pas faire cela tant que vous n'avez pas désactivé le mode "Invite Only".
     comments_helper:
       ama: ama
-      ask_me_anything: Ask Me Anything
-      author: Author
+      ask_me_anything: Demandez-moi n'importe quoi
+      author: Auteur
+      like: Aime
```

Comment:
Seems this is no longer used after fd690982c -- easily missed when merging in from main!

```suggestion
```